targets/TARGET_Freescale/mbed_rtx.h

```diff
@@ -133,4 +123,11 @@ extern uint32_t               Image$$ARM_LIB_STACK$$ZI$$Length[];
 
 #endif
 
+#if defined(__ARMCC_VERSION)
```

Comment:
Freescale files have LIB_HEAP section defined for all targets, but for other targets it is still missing.
This will be removed when all targets have ARM_LIB_HEAP defined and default implementation will switch to ARM_LIB_HEAP instead calculating heap from stack and IRAM1